Docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+#
+# Dockerfile for h2o
+#
+
+FROM alpine
+MAINTAINER Lars K.W. Gohlke <lkwg82@gmx.de>
+
+RUN apk update \
+    && apk upgrade \
+    # need for ocsp stapling \
+    && apk add -U perl openssl \
+    && apk add -U build-base \
+               cmake \
+               curl \
+               linux-headers \
+               zlib-dev \
+               ca-certificates curl wget \
```

Comment:
Yes it was. See the new Dockerfile in the separate repository: https://github.com/lkwg82/h2o.docker/blob/master/Dockerfile

Cheers Lars